orders/models.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from products.models import Base, Product
+from users.models import User
+
+class Cart(Base):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    product = models.ForeignKey(Product, on_delete=models.CASCADE)
+    quantity = models.PositiveIntegerField()
+
+    class Meta:
+        db_table = 'carts'
+
+class Order_Status(Base):
+    order_status = models.CharField(max_length=100)
+    
+    class Meta:
+        abstract = True
```

Comment:
- python에서 class명은 `CamelCase`로 만들어주세요.
- abstact =True 속성은 지워주세요!

```suggestion
class OrderStatus(Base):
    status = models.CharField(max_length=100)
    
    class Meta:
        db_table = 'order_status'
```